app/models/cart.rb

```diff
@@ -30,4 +30,12 @@ def subtotal(item_id, price)
     price * quantity_of(item_id)
   end
 
+  def number_of_days
+    @items['days'] ||= 0
+  end
+
+  def update_number_of_days(days)
+    @items['days'] = days.to_i
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.